arangodb-net-standard/CursorApi/ICursorApiClient.cs

```diff
@@ -35,11 +40,16 @@ public interface ICursorApiClient
                 long? memoryLimit = null,
                 int? ttl = null,
                 string transactionId = null,
+                CursorHeaderProperties headerProperties = null,
             CancellationToken token = default);
 
         /// <summary>
         /// Execute an AQL query, creating a cursor which can be used to page query results.
         /// </summary>
+        /// <remarks>
+        /// This method supports Read from Followers (dirty-reads). 
+        /// To enable it, set the AllowReadFromFollowers header property to true.
```

Comment:
Do we need to mention that this is from ArangoDB 3.10 onwards? Or was it available in previous versions?

Reply:
This is a feature introduced in 3.10. I've updated the comments.